environments/development.env

```diff
@@ -8,6 +8,7 @@
 
 NODE_ENV=development
 CONFIG_NAME=development
+CONFIG_ENV=development
```

Comment:
Do we need those 2 lines?

Reply:
debatable.   We only need it in prodbeta vs prod.  I wrote it all out explicitly though because I tried to automate it once and messed up.

But I think I will make that change again - if the only thing that needs it is prodbeta, then the only one who has it should be prodbeta.  We've already got too much duplication between NODE_ENV / CONFIG_NAME